### ISSUE TITLE
about ENLCA?

### ISSUE THREAD

**Issue by fanrz** [Zj-BinXia/ENLCA#3]
thanks for your work. I have on question.
in your paper, For each ordered sequence, we take the top n1N related ones as relevant features and n1N unrelated ones starting from n2N as irrelevant features.
u use
 `            positive = torch.mean(score[:, :, :5], dim=2)
            negative = torch.mean(score[:, :, 50:], dim=2)  # [N,H*W]`
why do you use 5 and 50?

**Comment by Zj-BinXia** [comment]
Sorry, we arrange the code after the paper being accepted. There may be the gap between the paper and code. You can change the code to "positive = torch.mean(score[:, :, :15], dim=2) negative = torch.mean(score[:, :, 50:65], dim=2) # [N,H*W] ".

**Comment by fanrz** [comment]
thank for your knid reply. 
I want to know for what reason, you pick 15 50 65.
any explanation for the values? or observation of the training?

**Comment by Zj-BinXia** [comment]
observation of the training

**Comment by fanrz** [comment]
great, thanks
